Framework/PythonInterface/plugins/algorithms/LoadWANDSCD.py

```diff
@@ -179,7 +302,41 @@ def PyExec(self):
         setGoniometer_alg.setProperty("Average", False)
         setGoniometer_alg.execute()
 
-        self.setProperty("OutputWorkspace", outWS)
+        return outWS
+
+    def normalize(
+        self,
+        data: IMDHistoWorkspace,
+        norm: IMDHistoWorkspace,
+        normalize_by: str,
+    ) -> IMDHistoWorkspace:
+        """
+        Normalize the given data with given Va normalization workspace
+        """
+        # prep
+        norm_replicated = ReplicateMD(ShapeWorkspace=data, DataWorkspace=norm)
+        data = DivideMD(LHSWorkspace=data, RHSWorkspace=norm_replicated)
+        # find the scale
+        if normalize_by == 'counts':
+            scale = norm.getSignalArray().mean()
+            self.getLogger().information('scale counts = {}'.format(int(scale)))
+        elif normalize_by == 'monitor':
+            scale = np.array(data.getExperimentInfo(0).run().getProperty('monitor_count').value)
+            scale /= norm.getExperimentInfo(0).run().getProperty('monitor_count').value[0]
+            self.getLogger().information('scale monitor = {}'.format(scale))
+        elif normalize_by == 'time':
+            scale = np.array(norm.getExperimentInfo(0).run().getProperty('duration').value)
```

Comment:
Should this be?

```suggestion
            scale = np.array(data.getExperimentInfo(0).run().getProperty('duration').value)
```

Reply:
These are the exact code from the original user story, which I assume is the one WAND2 is using at the moment.
I will confirm with the CIS to make sure that the example script in the user story is correct.

Reply:
Yes, that is a typo on my part.